Resources/Server Info/Guidebook/Chef/pitzah.xml

```diff
@@ -96,4 +103,102 @@
 <GuideEntityEmbed Entity="KitchenMicrowave" Caption="30 sec"/>
 <GuideEntityEmbed Entity="FoodPizzaDonkpocket" Caption=""/>
 </Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Mothic Margherita Pizza[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- Mozzarella
+- Cheese wedge
+- Ambrosia Vulgaris
+</Box>
+<Box Orientation="Vertical">
+- Dough
+- 2 Tomatoes
```

Comment:
I'd be up for simplicity and merge the normal Margherita recipe with this one. Maybe get rid of using Cheddar and stick to mozzie, remove Ambrosia, and be done?

Reply:
Would using the current amount of ingredients for the current Margherita work? 1 Mozzarella, 1 Dough, 4 Tomatoes.

Reply:
Hmm, 4 tomatoes is steep, I'm thinking 3 tomato and 2 mozzarella, it lines up better with typical harvests and mozzarella is made in pairs anyway.